src/app/my-box/my-box.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnInit } from '@angular/core';
+
+import { AddinClientService } from '@blackbaud/skyux-lib-addin-client';
+
+import { AddinClientInitArgs } from '@blackbaud/sky-addin-client';
+
+@Component({
+  selector: 'app-my-box',
+  templateUrl: './my-box.component.html',
+  styleUrls: ['./my-box.component.scss'],
+})
+export class MyBoxComponent implements OnInit {
+  public environmentId: string | undefined;
+  public context: string | undefined;
+  public userIdentityToken: string | undefined;
+
+  constructor(private addinClientService: AddinClientService) {}
+
+  public ngOnInit() {
+    this.addinClientService.args.subscribe((args: AddinClientInitArgs) => {
+      args.ready({
+        showUI: true,
+        title: 'Custom Box',
+        boxConfig: {
+          controls: [
+            {
+              name: 'Action 1',
+              action: 'action1',
+            },
+            {
+              name: 'Action 2',
+              action: 'action2',
+            },
+          ],
```

Comment:
You have action controls, but not handling them when they're clicked.  you can either remove them so we just don't demo the context menu, or provide a handler for them that indicates when they've been click (maybe toast messages)

Reply:
Added action handler, opening a toast message on action click. @michael-tims Thank you!

Reply:
Thanks!